recipes/aardvark-dns/conda_build_config.yaml

```diff
@@ -0,0 +1,15 @@
+c_compiler:
+  - gcc                        # [linux]
+  - clang                      # [osx]
+  - vs2022                     # [win]
+
+c_stdlib:
+  - sysroot                    # [linux]
+  - macosx_deployment_target   # [osx]
+  - vs                         # [win]
+
+c_stdlib_version:              # [unix]
+  - 2.17                       # [linux]
+  - 10.13                      # [osx and x86_64]
+  - 11.0                       # [osx and arm64]
+
```

Comment:
```suggestion
```
Please remove this file - these versions are handled automatically by conda-forge.